### ISSUE TITLE
No license file here either...

### ISSUE THREAD

**Issue by SimonBiggs** [secure-vote/sv-light-smart-contracts#13]

**Comment by XertroV** [comment]
Yeah, this is deliberate. It's unlicensed currently because we never got around to writing something. The code is up so it's publicly inspectable but not usable by E.g. governments

**Comment by SimonBiggs** [comment]
Kk, that does limit contributions however. I won't be making contributions without a declared robust license...

**Comment by XertroV** [comment]
That's cool - this isn't the part to make contributions to in any case (_this_ part is pretty much done). I am interested in figuring out licensing details for this and flux stuff tho. Haven't been able to prioritise going through ins and outs of each license or how to customise them (or even which ones we can).

**Comment by SimonBiggs** [comment]
Ideally you wouldn't customise them.

**Comment by XertroV** [comment]
> Ideally you wouldn't customise them.

yeah; but as I mentioned the code is pseudo-commercial and I'm not sure of a good solution that avoids it being libre software. I have heard of a thing "open core" that looks at first glace like it might help. Unsure though and haven't spent significant time looking into it for more than 12 months.

I have tried to segregate things a bit so we can publish as much as possible under another license (MIT or Apache or GPL or wahtever)
